### DISCUSSION TITLE
Is there support for multiple ENV vars to be displayed?

### DISCUSSION THREAD

**Discussion by 037g** [starship/starship#2215] [Q&A] [answered]
Hi,

I can't seem to be able to get multiple ENV vars to display. I've tried comma delimited and new sections.

`[env_var]
format = " [$env_value]($style)"
variable = "HTTP_PROXY,OTHER_ENV"
style = "bold red"
default = ""

[env_var]
format = " [$env_value]($style)"
variable = "OTHER_ENV"
style = "bold red"
default = ""
`

Thanks

**Comment by andytom** [comment, marked as answer]
Unfortunately this is not possible at the moment but we have an open issue #1194 to add this feature.

**Reply by 037g** [reply]
Got it, thank you.
